### DISCUSSION TITLE
Invert color of pages

### DISCUSSION THREAD

**Discussion by gmpike** [pdfarranger/pdfarranger#667] [Ideas]
It would be really useful, not just for reading purposes.
Printing a black background with white text consumes a lot of ink, inverting the colors solves the problem

**Comment by jeromerobert** [comment]
If this can be done by adding a single PDF command at the beginning of a page stream we could implement it. If this require to go through all objects of each pages then we can't. I'll check on simple uncompress PDF what those tools do: https://stackoverflow.com/questions/30284327/reverse-white-and-black-colors-in-a-pdf

@dreua Let's keep this under *discussion* until we decide if it goes to the TODO list or not.

**Comment by jeromerobert** [comment]
The postscript `{1 exch sub} settransfer` command works but according to `ps2pdf` it has no equivalent in PDF.

**Comment by dreua** [comment]
- Let's give this 3 weeks. If nothing productive comes up it should be closed and tagged `stale feature`.

**Comment by carafelix** [comment]
I would like for pdfarranger to have this functionality, specially for inverting only certain pages
